### DISCUSSION TITLE
Check-in with everyone

### DISCUSSION THREAD

**Discussion by dkershaw10** [microsoftgraph/msgraph-bicep-types#42] [General]
Hello folks,

Just checking-in to see how you are all getting on with trying out the private preview?

Thanks to folks who have been in touch or filed issues or responded on issues, and to everyone else who has given the private preview a try.  For those that haven't been able to yet, We're hoping that you can take a little time from your busy days to take a look and let us know what you think.  We'd love to hear from everyone.

We're looking to get more feedback on your usage experiences, whether the Microsoft Graph Bicep Extension has the potential to resolve existing painpoints for you, and what product gaps you are seeing that may prevent you using this capability in production once it goes to public preview.

We'll be sending out a short survey to everyone shortly to get feedback.

On behalf of the Microsoft Graph Bicep Extension team.

**Comment by WhitWaldo** [comment]
Just got access yesterday, but eager to give it a whirl later this week!
